README.md

```diff
@@ -5,12 +5,21 @@
 * install [ldc](https://github.com/Kitware/ldc)
 * install submodules via `git submodule update --init --recursive`
 
+### Running ingest
+
+You need an active SSH tunnel connection to nersc attached to the compose network.  After running docker-compose up, run this container, making sure to replace `<username>`.
+
+If you haven't already, [set up MFA on your NERSC account](https://docs.nersc.gov/connect/mfa/) (it's required for SSHing in).
+
+```bash
+docker run --rm -it --network nmdc-server_default --name tunnel kroniak/ssh-client ssh -o StrictHostKeyChecking=no -L 0.0.0.0:27017:mongo-loadbalancer.nmdc-runtime-dev.development.svc.spin.nersc.org:27017 <username>@dtn01.nersc.gov '/bin/bash -c "while [[ 1 ]]; do echo heartbeat; sleep 300; done"'
+```
```

Comment:
I just realized I never made a PR or merged the branch where I'd already written some docs for this. I'll make a PR and we can reconcile them.

Reply:
I intended this to be the reconciled documentation from #566 that included your explanation of why this is necessary and how to set up MFA.

The other parts should no longer be necessary.  I can bring back the "dev" argument in the docs if you want.

Reply:
Oh cool, forgot about the resolution there. Let's merge this.